tiatoolbox/wsicore/wsireader.py

```diff
@@ -709,7 +713,7 @@ def read_bounds(
         interpolation="optimise",
         pad_mode="constant",
         pad_constant_values=0,
-        location_is_at_requested=False,
+        location_at_requested=False,
```

Comment:
@shaneahmed  @vqdang I'm still not super keen on this kwarg name. What do you think? Just trying to make a good decision now as it's a pain to change it later. Alternative ideas:

```
baseline_coords=True
coords_space="baseline"  # Alternative being coords_space="resolution" or similar for coords at the resolution
```

Reply:
I would suggest `baseline_coords = True (default)` which will work opposite to `location_at_requested`.